js/CADWorker/CascadeStudioStandardUtils.js

```diff
@@ -73,16 +88,9 @@ function recursiveTraverse(x, callback) {
   }
 }
 
-/** This function returns a version of the `inputArray` without the `objectToRemove`. */
-function Remove(inputArray, objectToRemove) {
-  return inputArray.filter((el) => {
-    return el.hash !== objectToRemove.hash ||
-           el.ptr  !== objectToRemove.ptr;
-  });
-}
```

Comment:
Since this was only ever used to Mutate `sceneShapes` I put it along with `sceneShapes` in their own module `js/CADWorker/CascadeStudioSceneShapesService.js` to be imported elsewhere, but I also changed the function a little, see comment there.